core/node/genesis/src/lib.rs

```diff
@@ -413,6 +413,14 @@ pub async fn create_genesis_l1_batch(
     transaction
         .blocks_dal()
         .insert_l1_batch(
+            genesis_l1_batch_header.number,
+            genesis_l1_batch_header.timestamp,
+            BatchFeeInput::default(),
```

Comment:
`BatchFeeInput::default()` returns `L1Pegged` variant which is legacy one. I'm not sure if there is a reason why `default` does so. I would explicitly create `PubdataIndependent` variant.

Reply:
If you look at the start of the file we have been using `L1Pegged` even before this PR. I changed it to `PubdataIndependent` as suggested, lmk if that's what not what you intended.